packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -478,7 +480,8 @@ class Splitter extends (CollectionWidget as any) {
     this._resizeHandles.forEach((resizeHandle) => {
       const { disabled, separatorSize } = resizeHandle.option();
 
-      size += disabled ? INACTIVE_RESIZE_HANDLE_SIZE : separatorSize as number;
+      size += disabled ? INACTIVE_RESIZE_HANDLE_SIZE
```

Comment:
Let's move this logic to resizeHandle. We can create getSize() method in resizeHandle and use its. In this case INACTIVE_RESIZE_HANDLE_SIZE & DEFAULT_RESIZE_HANDLE_SIZE should be removed from this file.